css/fonts.css

```diff
@@ -1,28 +1,24 @@
-h1, h2, h3{
+@import url('https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&display=swap');
```

Comment:
imported font within styles: did not work, due to using the tool of w3-school (only kept as a test)

Reply:
this style sheet only works for the paragraph font - not headings so no changes were committed